proto/qdrant/cloud/account/v1/account.proto

```diff
@@ -418,6 +418,12 @@ message Account {
   // The privileges of the account, if any.
   // This is a read-only field.
   repeated string privileges = 8 [(buf.validate.field).repeated.items.string = {min_len: 1}];
+  // This field is read-only and is set by the server based on the authenticated user.
+  // It indicates whether the authenticated user is the owner of the account.
```

Comment:
please switch order of lines (on all places we mention read-only / required) not to start with...